src/jira/client/jira-client.ts

```diff
@@ -265,7 +267,7 @@ export const getJiraClient = async (
 					};
 				}
 				logger?.debug(`Sending builds payload to jira. Payload: ${payload}`);
-				logger?.info("Sending builds payload to jira.");
+				logger?.info({ gitHubProduct }, "Sending builds payload to jira.");
```

Comment:
Should we do this in the first line of this method when `logger` is created? 
`const logger = log.child({ jiraHost, gitHubInstallationId, gitHubProduct });`

So that we can save the changes in all other places?